en/drafts/originals/interactive-data-visualization-dashboard.md

```diff
@@ -422,137 +447,143 @@ line_fig_rus.update_layout(
+
+On the next screen, you will enter several pieces of information. In addition to giving your dashboard a name, you'll need to configure two more settings (assuming all the populated default settings are correct). 
 
-If your GitHub repository is public, you can copy and paste the HTTPS address of the repository into the address of "Public Git Repository." 
+First, change the **Start Command** input to `gunicorn app:server`. The `server` name after the colon should match the object name you set for your server in your Python script. The `app` name before the colon should match the `.py` filename in the repository.
 
-Otherwise, you can also link your GitHub account with Render so that Render has access to your private repository.
+Second, scroll down to find the section called **Environment Variables**. Click _Add Environment Variable_ and input `PYTHON_VERSION` as the key, with the Python version that you use on your machine as the value (use `$python -V` in the command line to check your Python version). If you don't explicitly specify the Python version this way, Render will use Python 3.7 as default, which may cause conflicts with the library versions you've specified in `requirements.txt`.
 
-Then, you will enter several pieces of information on the next screen. In addition to giving your dashboard a name, you need to configure two more settings (assuming all the populated default settings are correct). 
+Third, click _Create Web Service_ and wait for several minutes for the application to build. When finished, you'll be able to see your dashboard via a URL, like this: `https://ph-dash-demo.onrender.com/`.
 
-First, in "Start Command," change the input to `gunicorn app:server`. That is, the name after the colon must be the same as the object name of the server you set in your Python script. The name before the colon must be the same as the `.py` file in the repository.
+## Further Practice: Second Example
 
-Second, scroll down and find the section called "Environment Variables." Click "Add Environment Variable" and input `PYTHON_VERSION` as the key and the Python version that you use on your machine as the value (use `$python -V` in command line to check your Python version). If you don't explicitly specify the Python version this way, Render will use Python 3.7 as default, which may cause conflicts between this old Python version and the libraries' versions specified in `requirements.txt`.
+To demonstrate the wide applicability of the approach used in the case study above, this lesson will show you how to create another dashboard, using a different dataset. This second example explores a different research question: 'What were the top 10 languages used by non-English U.S. newspapers by decade from the 1690s to today?' We'll design a dashboard to show the top ten languages in each decade dating back to the 1690s, highlighting any shifts in their rankings, and the emergence or decline of different languages over time. 
```

Comment:
I tried changing the wording of the second research question here, but I still think it is very tricky to try to express your particular question in a clear way. Do you prefer the original? Maybe we can brainstorm some options...

Reply:
I think your version is better!